src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -411,6 +405,33 @@ private static Boolean parseBoolean(final String potentialArgumentValue)
         return null;
     }
 
+    private List<String> getBooleanTabCompletions(String potentialArgumentValue)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 8, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make "getBooleanTabCompletions" a "static" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2325)